.gitignore

```diff
@@ -126,3 +126,7 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+// files
+
+ui_testing/1lims-automation/*
```

Comment:
revert this changes
`git checkout master .gitignore && git add .gitignore && git commit -m 'fix' && git push`